Mathlib/Analysis/Complex/IntegerCompl.lean

```diff
@@ -0,0 +1,55 @@
+/-
+Copyright (c) 2024 Chris Birkbeck. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Birkbeck
+-/
+
+import Mathlib.Analysis.Complex.UpperHalfPlane.Basic
+
+/-!
+# Integer Complement
+
+We define the complement of the integers in the complex plane and give some basic lemmas about it.
+We also show that the upper half plane embeds into the integer complement.
+
+-/
+
+open UpperHalfPlane
+
+/--The complement of the integers in `ℂ`. -/
+abbrev Complex.IntegerComplement := (Set.range ((↑) : ℤ → ℂ))ᶜ
```

Comment:
This is a set, so it should use `lowerCamelCase`.
```suggestion
abbrev Complex.integerComplement := (Set.range ((↑) : ℤ → ℂ))ᶜ
```
BTW, why do you want to have it reducible?

Reply:
It should find it if you `rw [Complex.integerComplement]` and then use `apply?`. I suspect you don't want this to be an `abbrev`, but I could be wrong.

Reply:
Ah yes ok that works. I changed it back to a `def`. But out of curiosity, why do you think we don't want it to be an abbrev? (I dont have much of a feel for these things).

Reply:
I'm not totally sure that it should be `def` over `abbrev`; it's just my intuition here. I don't think there's anything strong pointing in either direction.